apprise/plugins/NotifyMailgun.py

```diff
@@ -184,11 +184,11 @@ def __init__(self, apikey, targets, from_name=None, region_name=None,
             self.logger.warning(msg)
             raise TypeError(msg)
 
-        if not VALIDATE_API_KEY.match(self.apikey):
-            msg = 'The API Key specified ({}) is invalid.' \
-                  .format(apikey)
-            self.logger.warning(msg)
-            raise TypeError(msg)
+#        if not VALIDATE_API_KEY.match(self.apikey):
```

Comment:
similar to the last comment; just go ahead and completely remove this; including the test case you found that it would have otherwise crapped out on.